tapir/coop/templates/coop/resignedmembership_detail.html

```diff
@@ -0,0 +1,109 @@
+{% extends "core/base.html" %}
+{% load django_bootstrap5 %}
+{% load i18n %}
+{% load utils %}
+{% load core %}
+{% load accounts %}
+{% block title %}
+    {% translate "Resigned member" %}: {% get_display_name_for_viewer object.share_owner request.user %}
+{% endblock title %}
+{% block content %}
+<div class="modal fade"
+    id="confirmDeleteModal"
+    tabindex="-1"
+    aria-labelledby="confirmDeleteModalLabel"
+    aria-hidden="true">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title">Attention!</h5>
+          <button type="button" class="close" data-bs-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+          <p class="m-2"><strong>Are you sure you want to remove {{ object.share_owner.user.first_name }} {{ object.share_owner.user.last_name }} from the resigned members list and to reactivate the member? This cannot be undone!</strong></p>
+        <div class="modal-footer">
+            <form class="form-inline ms-3"
+                method="post"
+                action="{% url 'coop:resign_member_remove' object.pk %}">
+                {% csrf_token %}
+                <button type="submit" class="{% tapir_button_custom 'warning' %}">
+                    <span class="material-icons">mood</span>Reactivate
+                </button>
+            </form>      
+            <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>    
+        </div>
+      </div>
+    </div>  
+</div>
+<div class="container">
+    <div class="card mb-2">
+        <h5 class="card-header">{% translate "Resigned member" %}: {% get_display_name_for_viewer object.share_owner request.user %}</h5>
+        <div class="card-body" id="resignedmember_detail_card">
+            <div class="row m-1">
+                <div class="col-3 fw-bold text-end">{% translate "Share owner" %}:</div>
+                <div class="col-9" id="resigendmember_share_owner">{% get_display_name_for_viewer object.share_owner request.user %}</div>
+            </div>
+            <div class="row m-1">
+                <div class="col-3 fw-bold text-end">{% translate "Cancellation reason" %}:</div>
+                <div class="col-9" id="resigendmember_cancellation_reason">{{ object.cancellation_reason }}</div>
+            </div>                
+            <div class="row m-1">
+                <div class="col-3 fw-bold text-end">{% translate "Cancellation date" %}:</div>
+                <div class="col-9" id="resigendmember_cancellation_date">{{ object.cancellation_date }}</div>
```

Comment:
This is displayed incorrectly:
![image](https://github.com/SuperCoopBerlin/tapir/assets/1138746/d474759a-93e1-40ad-b700-4519a7fa92ad)
Should be 28 / 06 / 2024.

Reply:
Fixed